Frontend/src/Components/Review/CreateReviewModal.jsx

```diff
@@ -0,0 +1,49 @@
+import React, { useState } from "react";
+import "./CreateReviewModal.css";
+
+const CreateReviewModal = ({ isOpen, onClose, onSubmit }) => {
+  const [rating, setRating] = useState(0);
+  const [content, setContent] = useState("");
+  const [successMessage, setSuccessMessage] = useState("");
+  const [errorMessage, setErrorMessage] = useState("");
+
+  if (!isOpen) return null;
+
+  const handleRatingClick = (index) => {
+    setRating(index + 1);
+  };
+
+  const handleSubmit = () => {
+    onSubmit({ rating, content }, setSuccessMessage, setErrorMessage);
+  };
+
+  return (
+    <div className="modalOverlay">
+      <div className="modalContent">
+        <h2>Leave a Review</h2>
+        <div className="rating">
+          {[...Array(5)].map((star, index) => (
```

Comment:
what does star represent here?

Reply:
oh I just made star as a placeholder, but i could change that to an underscore.

Reply:
yeah let's do that!